processor-api/src/main/java/com/synopsys/integration/alert/processor/api/filter/JobNotificationMapper.java

```diff
@@ -88,9 +88,9 @@ public JobNotificationMapper(JobAccessor jobAccessor) {
         FilteredDistributionJobRequestModel filteredDistributionJobRequestModel = new FilteredDistributionJobRequestModel(
             frequencyTypes,
             EnumUtils.getEnum(NotificationType.class, contentWrapper.extractNotificationType()),
-            detailedNotificationContent.getProjectName(),
+            detailedNotificationContent.getProjectName().orElse(null),
             detailedNotificationContent.getVulnerabilitySeverities(),
-            detailedNotificationContent.getPolicyNames()
+            detailedNotificationContent.getPolicyName().map(List::of).orElse(List.of())
```

Comment:
This is a different API that we can update separately.

Reply:
Just to be clear, DetailedNotificationContent is for one policy and FilteredDistributionJobRequestModel is for a list of policies?
Is that what you are referring to that we can update separately?

Reply:
Yes. The database API requests a `List`, but the processing API requests a `String`.